resources/tdm_gcc.rb

```diff
@@ -87,6 +87,19 @@
       overwrite true
     end
   end
+
+  # Patch time.h headers for compatibility with winpthreads.
+  # These patches were made for binutils 2.25.1 for 32-bit TDM GCC only.
```

Comment:
Sadly, none worthy of actually linking.  All my knowledge is from  trawling through a bunch of shitty mailing lists and old posts from 2011 or 2012 with people insulting each other over the internet.

The closest I have to a civil, yet not asinine source is https://lists.gnu.org/archive/html/emacs-devel/2013-06/msg00034.html (and its ancestors and children).  It's not entirely accurate - I haven't found the root cause but some change in msys-core-bin from 1.0.17 to 1.0.18 caused make to freak out and not accurately fork/reap child processes.

The only files provided by msys-core-bin are:
```
msysmnt.exe
ps.exe
error-mode.exe
msys-1.0.dll
```

So...  yeah...  someone in the core msys library, something trivial changed 3 or 4 years ago and nobody has bothered to fix it since...

One day we'll move to msys 2 and it'll be stable and things will be better.